ENG-1677 - Passwordless send phone rate limiting (#3773)
FusionAuth/fusionauth-site: astro/src/content/docs/apis/_tenant-request-body.mdx

  </APIField>

  <APIField name="tenant.rateLimitConfiguration.sendPasswordlessPhone.enabled" type="Boolean" optional defaults="false" since="1.99.9">

>     Whether rate limiting is enabled for send passwordless via phone.

Reviewer: Same comment as above.

```suggestion
    Whether rate limiting is enabled for the requesting of passwordless login links via SMS.
```

    <EnterprisePlanBlurbApi feature="rate limiting" />

  </APIField>

  <APIField name="tenant.rateLimitConfiguration.sendPasswordlessPhone.limit" type="Integer" optional defaults="5" since="1.99.9">

>     The number of times a user can request a passwordless login message within the configured <InlineField>timePeriodInSeconds</InlineField> duration.

Reviewer: ```suggestion
    The number of times a user can request a passwordless login SMS message within the configured <InlineField>timePeriodInSeconds</InlineField> duration.
```

    Required when `enabled` is set to `true`.

    <EnterprisePlanBlurbApi feature="rate limiting" />

  </APIField>

  <APIField name="tenant.rateLimitConfiguration.sendPasswordlessPhone.timePeriodInSeconds" type="Integer" optional defaults="60" since="1.99.9">

>     The duration for the number of times a user can request a passwordless login message before being rate limited.

Reviewer: ```suggestion
    The duration for the number of times a user can request a passwordless login SMS message before being rate limited.
```

Author: I can't make up my mind about how much to talk about SMS or not
